firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,126 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import (
+    By,
+    expected
+)
+from marionette.errors import (
+    NoSuchElementException,
+    StaleElementException
+)
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSafeBrowsingNotificationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = [
+            # Phishing URL info = mozmill 18Feb2015
+            {
+                'button_label': 'safebrowsing.notAForgeryButton.label',
+                'report_page': 'www.google.com/safebrowsing/report_error',
+                'unsafe_page': 'https://www.itisatrap.org/firefox/its-a-trap.html'
+            },
+            # Malware URL object  = mozmill 18Feb2015
+            {
+                'button_label': 'safebrowsing.notAnAttackButton.label',
+                'report_page': 'www.stopbadware.org',
+                'unsafe_page': 'https://www.itisatrap.org/firefox/its-an-attack.html'
+            }
+        ]
+
+        self.prefs.set_pref('browser.safebrowsing.enabled', True)
+        self.prefs.set_pref('browser.safebrowsing.malware.enabled', True)
+
+        # TODO? workaround for bug 1084289, mozmill: aModule.browserWindow.maximize()
+
+        # Give the browser a little time, because SafeBrowsing.jsm takes a
+        # while between start up and adding the example urls to the db.
+        # https://dxr.mozilla.org/mozilla-central/source/browser/base/content/browser.js#1194
+        # viewed 2015-02-24
```

Comment:
You can use mercurial to link to a certain changeset, i.e.: https://hg.mozilla.org/mozilla-central/file/46aebcd9481e/browser/base/content/browser.js#l1194